assets/chef-recipes/cookbooks/mdbe_build/recipes/default.rb

```diff
@@ -399,17 +426,28 @@
       command "sed -i~orig -e 's/# deb-src/deb-src/' /etc/apt/sources.list"
     end
   when 22.04 # Ubuntu Jammy
+    package %w(krb5-multidev libkrb5-dev) do
```

Comment:
Why these packages are not part of the corresponding distribution package list? Why it is important to install them separately?

Reply:
Then put the installation of build dependencies in Debian/Ubuntu distributions after installation of the general packages.